build/push.sh

```diff
@@ -0,0 +1,15 @@
+CHANGED_FILES_AMOUNT=$(git status -s | wc -l)
+# if more than one file modified => not only package.json changed
+if [ "$CHANGED_FILES_AMOUNT" -gt 1 ]
+then
+  # what is the current branch?
+  BRANCH=$(git branch --show-current)
```

Comment:
Can we move the branch above the if and add an "or branch is master" to the if?

Reply:
Has been done